src/applications/proxy-rewrite/utilities/get-target-env.js

```diff
@@ -0,0 +1,16 @@
+let targetEnv = '';
```

Comment:
This lives in the platform/site-wide folder so in the interest of Teamsites code isolation, it is duplicated here now.